packages/ladle/lib/cli/vite-plugin/generate/get-meta-json.js

```diff
@@ -32,6 +38,9 @@ const getMetaJson = (entryData) => {
       ...storyIdToMeta(storyId),
       ...storyMeta[storyId],
       meta: storyParams[storyId] ? storyParams[storyId].meta : {},
+      namedExports: namedExports[storyId]
```

Comment:
```suggestion
      namedExport: namedExports[storyId]
```

There is just single `namedExport` for each story?